### DISCUSSION TITLE
How do I tell vendure Admin UI the assetUrlPrefix where my assets are hosted when using a standalone Admin UI?

### DISCUSSION THREAD

**Discussion by motherofallcheetahs** [vendurehq/vendure#3163] [Q&A]
when using the AdminUI plugin:
adminUiConfig: {
        apiPort: serverPort,
        assetUrlPrefix: 'https://....',
      },

How about when deploying a stand-alone Admin UI?

**Comment by michaelbromley** [comment]
The asset prefix url is not controlled by the Admin UI - it is part of the AssetServerPlugin: https://docs.vendure.io/reference/core-plugins/asset-server-plugin/asset-server-options#asseturlprefix
